src/app/components/navigation-bar/navigation-bar.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-navigation-bar',
+  templateUrl: './navigation-bar.component.html',
+  styleUrls: ['./navigation-bar.component.css']
+})
+export class NavigationBarComponent {
+
+	constructor(private readonly router: Router) { }
+
+	public home(): void {
+		this.router.navigate(['/home']);
```

Comment:
could be just one method that accepts a string, or you could write directly in template: `router.navigate(['/home'])`, no need to create a method for each of the routes